Frontend/Frontend.Client/Components/MessageComponent.razor

```diff
@@ -13,10 +14,10 @@
 
 @code {
     [Parameter]
-    public string? User { get; set; }
+    public required string User { get; set; }
 
     [Parameter]
-    public string? MessageContent { get; set; }
+    public required string MessageContent { get; set; }
```

Comment:
This message component is only used for user messages so these two are now required rather than nullable